recipe/meta.yaml

```diff
@@ -7,6 +7,8 @@ package:
 source:
   url: https://github.com/GenericMappingTools/gmt/releases/download/{{ version }}/gmt-{{ version }}-src.tar.gz
   sha256: 2cd073d8b1f11b87a59b38aa7aaf8e39c7940d2b55a46a3267e23802ec8a08d1
+  patches:
+    - crosscompiling.patch
```

Comment:
The GMT v6.3.0 in the main branch needs a patch here.

Do we need to update the build number to `4`?

Reply:
From scanning some of the other ARM OSX Migration patches, I would say no. Maybe merge this first (easier to cherry-pick into the `dev` and `v5.x` branches), and then bump the build number in a follow-up commit?

Reply:
> easier to cherry-pick into the `dev` and `v5.x` branches

The same patch has been merged into the GMT master branch, so for the `dev` branch, you need to update it to the latest commit and then let the bot re-run the ARM OSX Migrator.

For `v5.x` branch, I think there were some changes in these files and I may make a separate patch later.

Reply:
In that case, it should be ok to do the version bump here then.